Parsers/Tell a Random Joke.js

```diff
@@ -2544,4 +2544,4 @@ while (!selectedJoke){
 var sendIt = new x_snc_slackerbot.Slacker().send_chat(current, selectedJoke.joke, false);
 
 //Send the punchline
-var sendItTwoElectricBoogallo = new x_snc_slackerbot.Slacker().send_chat(JSON.parse(sendIt.getBody()), selectedJoke.punchline, !sendIt.haveError());
\ No newline at end of file
+var sendItTwoElectricBoogallo = new x_snc_slackerbot.Slacker().send_chat(JSON.parse(sendIt.getBody()), selectedJoke.punchline, !sendIt.haveError());
```

Comment:
no actual changes? why is this being included in the PR?

Reply:
This one caused a merge conflict for some reason - just left it the same way it was in `main`